rules/aip0133/resource_reference_type.go

```diff
@@ -29,19 +29,19 @@ var resourceReferenceType = &lint.MethodRule{
 		// Return type of the RPC.
 		ot := m.GetOutputType()
 		if ot.GetName() == "Operation" {
-			ot = utils.GetResponseType(m)
+			ot = utils.GetOperationResponseType(m)
 		}
```

Comment:
I realize this isn't the diff, but we need to clean this up, because this isn't a robust enough check. Compute and Cloud SQL, while not compliant so they might not have api-linter running, define their own `Operation` type and are not using LRO. This should be consolidated into the more robust `utils.GetResponseType(m)`. There are likely other spots, I think we need a fixit :)

Reply:
ack. Here's a tracking issue: https://github.com/googleapis/api-linter/issues/1154